docs/client/connecting-kn-to-your-cluster.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Connecting the `kn` Client to your cluster"
+linkTitle: "Connecting to your cluster"
+weight: 10
+type: "docs"
+---
+
+The `kn` Client uses your `kubectl` client configuration, the kubeconfig file, to connect to your cluster.
```

Comment:
Looks like there's a weird new line between 8-9. Can you delete that? (May have been my fault from using the "suggestions" feature, so sorry about that!